ksql-parser/src/test/java/io/confluent/ksql/parser/KsqlParserTest.java

```diff
@@ -1234,4 +1236,46 @@ public void shouldParseMultiLineWithInlineBracketedComments() {
     assertThat(statements, hasSize(1));
     assertThat(statements.get(0).getStatement(), is(instanceOf(ListStreams.class)));
   }
+
+  @Test
+  public void shouldBuildSimpleCaseStatement() {
```

Comment:
```suggestion
  public void shouldBuildSearchedCaseStatement() {
```
Rename to avoid confusion if we want to support simple case expressions in the future.